apps/authenticator/lib/sign_out/commands/inputs/revoke_tokens.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Authenticator.SignOut.Commands.Inputs.RevokeTokens do
+  @moduledoc """
+  Input schema to be used in Revoke Tokens flow.
+  """
+
+  use Authenticator.Input
+
+  @typedoc "Revoke Tokens flow input fields"
+  @type t :: %__MODULE__{
+          access_token: String.t() | nil,
+          refresh_token: String.t() | nil
```

Comment:
```suggestion
          access_token: String.t(),
          refresh_token: String.t()
```

These fields are required

Reply:
Actually it's not mandatory anymore.